docs/deploy-options.md

```diff
@@ -162,15 +157,20 @@ curl -H 'Host: kuard.local' ${CONTOUR_IP}
 
 ## Running without a Kubernetes LoadBalancer
 
-If you can't or don't want to use a Service of `type: LoadBalancer` there are two alternate ways to run Contour.
+If you can't or don't want to use a Service of `type: LoadBalancer` there are other ways to run Contour.
 
 ### Local development
 
-As mentioned above, when you are doing local development, you can run a local cluster with [minikube](https://kubernetes.io/docs/setup/minikube/) or [kind](https://github.com/kubernetes-sigs/kind). Follow the instructions under [Get your hostname or IP address][5] to get access, then use your user agent of choice.
+As mentioned above, when you are doing local development, you can run a local cluster with [minikube](https://kubernetes.io/docs/setup/minikube/) or [kind](https://github.com/kubernetes-sigs/kind).
+Follow the instructions under [Get your hostname or IP address](#get-your-hostname-or-ip-address) to get access, then use your user agent of choice.
```

Comment:
Does this apply when running locally? Didn't the minikube and kind sections just cover this? Maybe this isn't necessary.